### PR TITLE
Add IWantMyName

### PR DESCRIPTION
**Pull request by Anshuman-Verma**
**https://iwantmyname.com/**

**Helps you search domains across all TLDs.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] The app I added is not a duplicate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,6 +224,7 @@ See [plotting graphs examples](https://www.wolframalpha.com/examples/PlottingAnd
 * [Emoji Search Online](http://www.webpagefx.com/tools/emoji-cheat-sheet/) - Intelligent emoji search engine.
 * [Podcast Feed Extractor](http://itunes.so-nik.com/) - Get RSS feed for podcasts.
 * [RSS Feed Search](http://ctrlq.org/rss/) - Find RSS feeds from search terms.
+* [IWantMyName](https://iwantmyname.com/) - helps you search domains across all TLDs.
```
Comment (aviaryan) [inline README.md]: Make the first letter in description capital. ("H")

Comment (Anshuman-Verma) [inline README.md]: Fixed

**Comment by Anshuman-Verma** [comment]
Done as requested.

**Comment by Anshuman-Verma** [comment]
Updated comment and description. :man_facepalming:

**Pull request merged**